### PR TITLE
fix: Fix ep.PythonCmd return values.

### PR DESCRIPTION
**Pull request by zhanluxianshen**
Fix ep.PythonCmd return values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ func main() {
 		panic(err)
 	}
 
-	cmd := ep.PythonCmd("-c", "print('hello')")
+	cmd, ise := ep.PythonCmd("-c", "print('hello')")
```
Comment (codablock) [inline README.md]:
```suggestion
	cmd, err := ep.PythonCmd("-c", "print('hello')")
```

File: README.md [hunk 2]
```diff
@@ -20,7 +20,10 @@ func main() {
...
+	if ise != nil {
```
Comment (codablock) [inline README.md]:
```suggestion
	if err != nil {
```

**Review by codablock — COMMENTED** [review]
@zhanluxianshen Thanks for the fix. I added 2 suggestions regarding naming of the `err` variable :) If you add these, I'll merge.

**Comment by zhanluxianshen** [comment]
thanks for quick review.
your suggestion is ok for me.

**Comment by zhanluxianshen** [comment]
I will rebase it ..
Seems error.

**Comment by zhanluxianshen** [comment]
@codablock 
Now, its ok for merge.

**Pull request merged**